### PR TITLE
Assignment rules

### PR DESCRIPTION
**Pull request by mhussaincgi**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# List high-level changes as bullet items

Below are some examples

- [ ] Changed field permissions
- [ ] changed field data-type

# Deployment Tracker

List all the metadata that is pushed in this commit/PR. Full URL should be fine.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by cgijeffolsen — APPROVED** [review]
Approved, the client will validate this in UAT @mhussaincgi 
cc @NataliaNikishina @lovychauhanCGI

**Review by cgijeffolsen — APPROVED** [review]

File: dev-app-post/main/default/assignmentRules/Case.assignmentRules-meta.xml [hunk 1]
```diff
@@ -54,5 +54,102 @@
             </criteriaItems>
             <template>unfiled$public/SUPPORTNewassignmentnotificationSAMPLE</template>
         </ruleEntry>
+        <ruleEntry>
+            <assignedTo>Regular</assignedTo>
+            <assignedToType>Queue</assignedToType>
+            <criteriaItems>
+                <field>Case.RecordTypeId</field>
```
Comment (yashrohan-cgi) [inline dev-app-post/main/default/assignmentRules/Case.assignmentRules-meta.xml:61]: Shouldnt it be recordType.Name?

Comment (mhussaincgi) [reply dev-app-post/main/default/assignmentRules/Case.assignmentRules-meta.xml:61]: @yashrohan-cgi  The ticket is to pull the changes in the branch from Prod. No changes are done in the rule.

**Review by yashrohan-cgi — APPROVED** [review]

**Review by NataliaNikishina — APPROVED** [review]
@mhussaincgi since the build check errors have been resolved, i am approving the PR. Thanks!

**Pull request merged**